src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -316,11 +318,8 @@ int main(int argc, char * argv[])
             goto exit;
         }
 
-        // Wait for response until the Message interval.
-        while (!MessageIntervalExpired())
-        {
-            DriveIO();
-        }
+        // Suspend current thread during gMessageIntervalSeconds, or until notified (if the latter happens first).
```

Comment:
I believe this comment just restatest the code (both here and below). I would suggest removing it.

Reply:
Acked